ERCS/erc-7579.md

```diff
@@ -218,9 +218,8 @@ Smart accounts MAY implement a fallback function that forwards the call to a fal
 
 If the smart account has a fallback handler installed, it:
 
-- MUST implement authorization control
-- MUST use `call` to invoke the fallback handler
-- MUST utilize [ERC-2771](./eip-2771.md) to add the original `msg.sender` to the `calldata` sent to the fallback handler
+- MUST use `call` or `staticcall` to invoke the fallback handler
+- MUST utilize [ERC-2771](./eip-2771.md) to add the original `msg.sender` to the `calldata` sent to the fallback handler. Authorization control MAY be implemented by the fallback handler
```

Comment:
Imo `Authorization control MAY be implemented by the fallback handler` would fit better into the `Fallback Handlers` section under `Modules` since thats where behaviours that modules should implement are defined

Reply:
you mean under Module Configurations right?
section where above comment is in this review

Reply:
reverted this and added at recommended place.